langchain/src/embeddings/hf_transformers.ts

```diff
@@ -1,4 +1,4 @@
-import { TransformersEmbeddingFunction } from "chromadb";
+import { Pipeline, pipeline } from "@xenova/transformers";
```

Comment:
This will be breaking (though it was undocumented before so not super concerned about that), and we'll want to add it to the docs and as a peer dep.

Reply:
Oh fascinating, the docs were wrong. Odd that this is working at all!

Reply:
Ohhh never mind this branch is off a branch! Let's just rebase it to `main`.